src/components/DarkModeToggle/DarkModeToggle.styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+import { sm } from '../../utils/constants/responsive-sizes';
+
+import { AlignMiddle } from '../../utils/styled/helpers';
+import { ToggleOff, ToggleOn } from '../../utils/styled/icons';
+
+const iconEffects = `
+  margin-right: 0.2em;
+`;
+
+const Container = styled(AlignMiddle)`
+  @media screen and (max-width: ${sm}) {
+    display: none;
+  }
```

Comment:
Great 👍 , developing with different screen sizes in mind makes things easier in the long run